widgets/src/main/java/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -41,6 +41,7 @@ public class PentahoTab extends SimplePanel {
   private Widget content;
   protected Label label = new Label();
   private boolean solutionBrowserShowing;
+  private boolean closeable;
 
   public PentahoTab( String text, String tooltip, PentahoTabPanel tabPanel, Widget content, boolean closeable ) {
     this.content = content;
```

Comment:
I believe you should remove the recently added `focus-visible` class (line 50) (by @hv-vamsi in https://github.com/pentaho/pentaho-commons-gwt-modules/pull/824).

https://github.com/pentaho/pentaho-commons-gwt-modules/blob/0d000bb0f2493565ec17e04a2aaf842c9a872965/widgets/src/main/java/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java#L49

As well as a similar reference in https://github.com/hv-vamsi/pentaho-commons-gwt-modules/blob/00561f6ae56b4ddc4f77728a6dda1690d0e8d665/widgets/src/main/java/org/pentaho/gwt/widgets/client/buttons/ThemeableImageButton.java#L105.

Reply:
Done